### PR TITLE
added ngenius webhook endpoint

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: Api/Controllers/AgentControllers/PaymentsController.cs [hunk 1]
```diff
@@ -133,6 +134,19 @@ public async Task<IActionResult> PayByNGenius(string referenceCode)
...
 
 
+        /// <summary>
+        ///     NGenius webhook
+        /// </summary>
+        [HttpPost("accommodations/bookings/ngenius/webhook")]
+        [AllowAnonymous]
+        public async Task<IActionResult> NGeniusWebhook([FromBody] JsonDocument document)
+        {
+            await _nGeniusPaymentService.WebHook(document);
```
Comment (mingazhev) [inline Api/Controllers/AgentControllers/PaymentsController.cs:144]: ProcessWebHook maybe?

File: Api/Controllers/AgentControllers/PaymentsController.cs [hunk 2]
```diff
@@ -133,6 +134,19 @@ public async Task<IActionResult> PayByNGenius(string referenceCode)
         }
 
 
+        /// <summary>
+        ///     NGenius webhook
+        /// </summary>
+        [HttpPost("accommodations/bookings/ngenius/webhook")]
+        [AllowAnonymous]
+        public async Task<IActionResult> NGeniusWebhook([FromBody] JsonDocument document)
```
Comment (mingazhev) [inline Api/Controllers/AgentControllers/PaymentsController.cs:142]: looks more like a request, not a document

**Review by mingazhev — APPROVED** [review]
May be merged after comments fix

File: Api/Controllers/AgentControllers/PaymentsController.cs (hunk 1 above)
Comment (DFSko) [reply Api/Controllers/AgentControllers/PaymentsController.cs]: Fixed

File: Api/Controllers/AgentControllers/PaymentsController.cs (hunk 2 above)
Comment (DFSko) [reply Api/Controllers/AgentControllers/PaymentsController.cs]: Fixed

**Pull request merged**